aleph/search/facet.py

```diff
@@ -93,12 +93,17 @@ def update(self, result, key):
         result["label"] = key if event is None else event.title
 
 
-class DateFacet(Facet):
```

Comment:
Are there any compatibility issues with removing this DateFacet? Is there a chance that someone might be relying on this and that it might break their code?

Reply:
It hasn't been released yet and I don't think the frontend needs it any more since we switched to histograms for all date type fields. Correct me if I'm wrong here @kjacks

Reply:
Correct :) It's no longer needed by the ui